network-api/networkapi/fellows/templates/fellows_home.html

```diff
@@ -1,5 +1,161 @@
+    <p class="small-gray">description description description description description description description description description</p>
+    <div class="person-social-links mt-3">
+      <a href="" class="twitter gray small mr-4"></a>
+      <a href="" class="linkedIn gray small mr-4"></a>
+    </div>
+  </div>
+  <div class="featured-person col-6 col-sm-4 col-md-3 mb-4 mb-sm-0" key={index}>
+    <img class="img-fluid d-block" src={item.image} alt="Headshot"/>
+    <h2 class="h5-black my-2">Firstname Lastname</h2>
+    <p class="small-gray">description description description description description description description description description</p>
+    <div class="person-social-links mt-3">
+      <a href="" class="twitter gray small mr-4"></a>
+      <a href="" class="linkedIn gray small mr-4"></a>
+    </div>
+  </div>
+  <div class="featured-person col-6 col-sm-4 col-md-3 mb-4 mb-sm-0" key={index}>
+    <img class="img-fluid d-block" src={item.image} alt="Headshot"/>
+    <h2 class="h5-black my-2">Firstname Lastname</h2>
+    <p class="small-gray">description description description description description description description description description</p>
+    <div class="person-social-links mt-3">
+      <a href="" class="twitter gray small mr-4"></a>
+      <a href="" class="linkedIn gray small mr-4"></a>
+    </div>
+  </div>
+  <div class="col-md-12 text-center my-5">
+    <button class="btn btn-ghost">See Fellows Directory</button>
+  </div>
+</div>
+
+<div class="row mb-5">
+  <div class="col-md-12 my-5">
+    <hr class="hr-gradient mt-0">
+    <h4 class="h2-headings-white">Why Become a Fellow?</h4>
+  </div>
+  <div class="col-md-6">img</div>
+  <div class="col-md-6">
+    <p>Why Become a Fellow Why Become a Fellow Why Become a Fellow Why Become a Fellow</p>
+    <ul>
+      <li>Apple pie cotton candy lollipop lollipop lollipop topping.</li>
+      <li>Lemon drops. Tootsie roll dessert cake chocolate ice cream topping muffin.</li>
+      <li>Sugar plum caramels jelly.</li>
+      <li>Marshmallow jelly beans sesame snaps fruitcake jelly-o icing bear claw dessert.</li>
+    </ul>
+  </div>
+</div>
+
+<div class="row mb-5">
+  <div class="col-md-12 my-5">
+    <hr class="hr-gradient mt-0">
+    <h4 class="h2-headings-white">Expectations</h4>
+  </div>
+  <div class="col-md-6">
+    <p>Expectations Expectations Expectations Expectations Expectations Expectations Expectations Expectations Expectations Expectations Expectations Expectations</p>
+    <ul>
+      <li>Apple pie cotton candy lollipop lollipop lollipop topping.</li>
+      <li>Lemon drops. Tootsie roll dessert cake chocolate ice cream topping muffin.</li>
+      <li>Sugar plum caramels jelly.</li>
+      <li>Marshmallow jelly beans sesame snaps fruitcake jelly-o icing bear claw dessert.</li>
+    </ul>
+  </div>
+  <div class="col-md-6">img</div>
+</div>
+
+<div class="row py-5 px-md-2">
+  <div class="col-md-2">img</div>
+  <div class="col-md-10">
```

Comment:
A lot of chunks in the body block can probably be rendered from an inclusion tag (kinda like setting up a component)

Although these last two chunks of html are almost the same cept the top one (block this comment in line attached to) has `col-md-2` then `col-md-10` and the last one just under this comment has `col-md-1` and `col-md-11` which is the type of minor variance that makes things like an inclusion tag a bit annoying.

Reply:
I'm also not sure if any of these inclusion tags would be used outside of this page or not, or just this page, and if so, does jdnago have a way to do it in a light way way.